hw1/02-palindrome.js

```diff
@@ -1,3 +1,40 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
```

Comment:
style guide expects handleInput to be used after the function it is referencing as an argument